contrib/sarplus/python/pysarplus/__init__.py

```diff
@@ -1,2 +1,20 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from pathlib import Path
+
 from .SARModel import SARModel
 from .SARPlus import SARPlus
+
+__title__ = "pysarplus"
+__version__ = (Path(__file__).resolve().parent.parent / "VERSION").read_text().strip()
+__author__ = "Markus Cozowicz"
```

Comment:
I would change the author to `RecoDev Team at Microsoft` as we have in the rest of the repo https://github.com/microsoft/recommenders/blob/main/recommenders/__init__.py#L6.

The particular contributions of each developer are showed here: https://github.com/microsoft/recommenders/blob/main/AUTHORS.md

Reply:
Also @simonzhaoms please add your name to AUTHORS.md

Reply:
Done